azblob/url_service.go

```diff
@@ -38,6 +38,11 @@ func (s ServiceURL) GetUserDelegationCredential(ctx context.Context, info KeyInf
 	return NewUserDelegationCredential(strings.Split(s.client.url.Host, ".")[0], *udk), nil
 }
 
+func (s ServiceURL) GetAccountInfo(ctx context.Context) (*ServiceGetAccountInfoResponse, error) {
```

Comment:
>GetAccountInfo [](start = 20, length = 14)

this API works on container and blob too

Reply:
https://docs.microsoft.com/en-us/rest/api/storageservices/get-account-information#request